test/integration/index.js

```diff
@@ -15,15 +15,10 @@ describe('INTEGRATION TESTS', function () {
   require('./IndividualClaims');
   require('./YieldTokenIncidents');
   require('./Cover');
-
-  // TODO: reenable
-  // require('./Master');
+  require('./Master');
   // require('./PooledStaking');
-  // require('./Pool');
-  // require('./MCR');
-  // require('./MemberRoles');
-  // require('./Claims');
-  //
-  // require('./Gateway');
+  require('./Pool');
+  require('./MCR');
+  require('./MemberRoles');
   // require('./TokenController');
```

Comment:
why not enable TC as well?

Reply:
It has empty tests right now but yeah i can toggle it on